src/main/scala/iog/psg/cardano/CardanoApi.scala

```diff
@@ -76,166 +83,164 @@ trait CardanoApi {
+    * @return estimate fee request
+    */
+  def estimateFee(
+    fromWalletId: String,
+    payments: Payments,
+    withdrawal: Option[String],
+    metadataIn: Option[TxMetadataIn] = None
+  ): Future[CardanoApiRequest[EstimateFeeResponse]]
 
   /**
-   * Select coins to cover the given set of payments.
-   * Api Url: [[https://input-output-hk.github.io/cardano-wallet/api/edge/#tag/Coin-Selections #CoinSelections]]
-   *
-   * @param walletId wallet's id
-   * @param payments A list of target outputs ( address, amount )
-   * @return fund payments request
-   */
-  def fundPayments(walletId: String,
-                   payments: Payments): Future[CardanoApiRequest[FundPaymentsResponse]]
+    * Select coins to cover the given set of payments.
+    * Api Url: [[https://input-output-hk.github.io/cardano-wallet/api/edge/#tag/Coin-Selections #CoinSelections]]
+    *
+    * @param walletId wallet's id
+    * @param payments A list of target outputs ( address, amount )
+    * @return fund payments request
+    */
+  def fundPayments(walletId: String, payments: Payments): Future[CardanoApiRequest[FundPaymentsResponse]]
 
   /**
-   * Get transaction by id.
-   * Api Url: [[https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/getTransaction #getTransaction]]
-   *
-   * @param walletId wallet's id
-   * @param transactionId transaction's id
-   * @return get transaction request
-   */
+    * Get transaction by id.
+    * Api Url: [[https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/getTransaction #getTransaction]]
+    *
+    * @param walletId wallet's id
+    * @param transactionId transaction's id
+    * @return get transaction request
+    */
   def getTransaction[T <: TxMetadataIn](
-                                         walletId: String,
-                                         transactionId: String): CardanoApiRequest[CreateTransactionResponse]
+    walletId: String,
+    transactionId: String
+  ): CardanoApiRequest[CreateTransactionResponse]
 
   /**
-   * Update Passphrase
-   * Api Url: [[https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/putWalletPassphrase #putWalletPassphrase]]
-   * @param walletId wallet's id
-   * @param oldPassphrase current passphrase
-   * @param newPassphrase new passphrase
-   * @return update passphrase request
-   */
-  def updatePassphrase(
-                        walletId: String,
-                        oldPassphrase: String,
-                        newPassphrase: String): Future[CardanoApiRequest[Unit]]
+    * Update Passphrase
+    * Api Url: [[https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/putWalletPassphrase #putWalletPassphrase]]
+    * @param walletId wallet's id
+    * @param oldPassphrase current passphrase
+    * @param newPassphrase new passphrase
+    * @return update passphrase request
+    */
+  def updatePassphrase(walletId: String, oldPassphrase: String, newPassphrase: String): Future[CardanoApiRequest[Unit]]
```

Comment:
Can we keep the original format here too. (without the extra blank line)

Reply:
ok just pushed commit with:
`docstrings.style = "Asterisk"`

Reply:
I think we mis communicated here.  I mean keeping the original multi line style of the updatePassphrase declaration. I'm not sure what docstrings.style did here, but I've no opinion the difference in the javadoc style.

Reply:
hmm sure, maxColumn I can set from 120 to 110 and will put those arguments in new lines

Reply:
@mcsherrylabs check now :)